src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -500,27 +507,29 @@ const WorkflowView = _.flow(
     Utils.withBusyState(v => this.setState({ updatingConfig: v }))
   )(async newSnapshotId => {
     const { ajax: { Methods } } = this.props
-    const { modifiedConfig: { methodRepoMethod: { methodNamespace, methodName } } } = this.state
+    const { modifiedConfig: { methodRepoMethod: { methodNamespace, methodName } }, isRedacted } = this.state
     const config = await Methods.template({ methodNamespace, methodName, methodVersion: newSnapshotId })
     const modifiedInputsOutputs = await Methods.configInputsOutputs(config)
-    this.setState({ modifiedInputsOutputs: this.sortOptionalInputs(modifiedInputsOutputs) })
+    this.setState({ modifiedInputsOutputs: this.sortOptionalInputs(modifiedInputsOutputs), wasRedacted: isRedacted, isRedacted: false })
```

Comment:
I think that `isRedacted` and `wasRedacted` are confusing names. The distinction between the two is unclear.